src/Database/Attributes/Relation.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Winter\Storm\Database\Attributes;
+
+use Attribute;
+
+#[Attribute(Attribute::TARGET_METHOD)]
+class Relation
+{
+}
```

Comment:
This attribute can be used instead of a Relation class return type to indicate a method is a relation method.

It could be used in cases where a relation is dynamic or if, for some reason, the Dev doesn't want to use return types.